test/indexedDBUtils.test.js

```diff
@@ -0,0 +1,15 @@
+const indexedDBUtils = require('../src/indexedDBUtils')
+
+window.indexedDB = require('fake-indexeddb')
+
+test('DB should exist', async () => {
+  expect.assertions(1)
```

Comment:
You should not need to expect.assertion, this is required for asynchronous tests only.

Reply:
done